### PR TITLE
Document repo.metadata format

### PR DESCRIPTION
**Pull request by d3dave**

### CODE REVIEW

**Comment by ovirt-infra** [comment]
The commiter is not a member of organization nor whitelist. Please contact maintainer for patch verification.

**Comment by ovirt-infra** [comment]
The commiter is not a member of organization nor whitelist. Please contact maintainer for patch verification.

**Comment by ovirt-infra** [comment]
The commiter is not a member of organization nor whitelist. Please contact maintainer for patch verification.

**Comment by ovirt-infra** [comment]
The commiter is not a member of organization nor whitelist. Please contact maintainer for patch verification.

**Comment by ovirt-infra** [comment]
The commiter is not a member of organization nor whitelist. Please contact maintainer for patch verification.

**Comment by ovirt-infra** [comment]
The commiter is not a member of organization nor whitelist. Please contact maintainer for patch verification.

**Comment by ovirt-infra** [comment]
The commiter is not a member of organization nor whitelist. Please contact maintainer for patch verification.

**Comment by ovirt-infra** [comment]
The commiter is not a member of organization nor whitelist. Please contact maintainer for patch verification.

**Comment by ovirt-infra** [comment]
The commiter is not a member of organization nor whitelist. Please contact maintainer for patch verification.

**Comment by ovirt-infra** [comment]
The commiter is not a member of organization nor whitelist. Please contact maintainer for patch verification.

**Comment by ovirt-infra** [comment]
The commiter is not a member of organization nor whitelist. Please contact maintainer for patch verification.

**Comment by ovirt-infra** [comment]
The commiter is not a member of organization nor whitelist. Please contact maintainer for patch verification.

**Comment by ovirt-infra** [comment]
The commiter is not a member of organization nor whitelist. Please contact maintainer for patch verification.

**Comment by ifireball** [comment]
ci 👍

**Comment by nvgoldin** [comment]
ci test please

File: docs/Templates.rst [hunk 1]
```diff
@@ -33,5 +33,42 @@ Available templates
...
+members:
+
+    ``name``: 
+        The name of the repository.
+    
+    ``sources``: 
+        
+        ``<name>``: Name of a source.
+        
+            *TO-DO: no docs yet...*
```
Comment (gbenhaim) [inline docs/Templates.rst]: For remote source
```
type: http
args:
    baseurl: url to the root of the repo
```
For local source
```
type: file 
args:
    root: path to the root of the repo
```

File: docs/Templates.rst [hunk 2]
```diff
@@ -33,5 +33,42 @@ Available templates
...
+                        
+                    ``timestamp``:
+                        Creation time of the template version.
+                        
+                    ``handle``:
+                        Either a base file name of the template located in the
+                        root directory of the repository, or a root-relative
+                        path to the template file.
+
 .. _here: http://templates.ovirt.org/repo/
```
Comment (gbenhaim) [inline docs/Templates.rst]: I suggest adding a link to an example repo.metadata file:
http://templates.ovirt.org/repo/repo.metadata

Comment (d3dave) [inline docs/Templates.rst]: added

**Comment by nvgoldin** [comment]
ci test please

**Comment by gbenhaim** [comment]
@d3dave can you please rebase?

**Comment by d3dave** [comment]
@gbenhaim done

**Comment by gbenhaim** [comment]
ci test please

**Comment by gbenhaim** [comment]
ci merge please

**Pull request merged**